### DISCUSSION TITLE
认清自己是不是基本盘。。

### DISCUSSION THREAD

**Discussion by zhenglu666** [WeNeedHome/SummaryOfLoanSuspension#365] [Ideas]
恕我直言，在这发声的从来就不是执政的基本盘，执政的基本盘在农村，在更穷的人那里，大家都只是拉拢争取的小布尔乔亚而已。。。别看中国这么多网名，天天吵吵吵得能有多少。。。

**Comment by zhaomingtao-fighting** [comment]
这里也有农民的儿女一样。

**Comment by yalo-ddaa** [comment]
差不多得了，但凡你看一眼地方政府的财政收入，看看多少是靠卖地才赚来的。
有些地方房地产占了地方财政的一半以上，这不叫基本盘叫什么？

**Reply by oldCassadin** [reply]
他的意思是，颠覆性的力量只能来自于那些吃不饱穿不暖，更没有功夫上网的人。 有功夫在这叭叭的，通常被引导，分化，最终都会轻易的妥协，没有几个人敢真的豁出去的。

**Reply by Jesse3692** [reply]
没有革命性

**Reply by Jesse3692** [reply]
![image](https://user-images.githubusercontent.com/24291671/178946409-a0735df2-6521-43b1-bc22-daf9d3d31a38.png)

**Reply by lisongrui** [reply]
> 没有革命性

小布尔乔亚是这样 从以前案例来看 甚至有人抱着希望 等十年 等二十年 直到去世

**Reply by wedxz** [reply]
> 他的意思是，颠覆性的力量只能来自于那些吃不饱穿不暖，更没有功夫上网的人。 有功夫在这叭叭的，通常被引导，分化，最终都会轻易的妥协，没有几个人敢真的豁出去的。

这是事实,只要没有涉及到生存问题,中国人还是很能忍的

**Reply by zcaaron** [reply]
不是冷兵器时代了。

**Reply by zxcs001** [reply]
> 他的意思是，颠覆性的力量只能来自于那些吃不饱穿不暖，更没有功夫上网的人。 有功夫在这叭叭的，通常被引导，分化，最终都会轻易的妥协，没有几个人敢真的豁出去的。

你去看看这两月，抖音、头条、西瓜视频上，各种评论是怎么翻车的。

这帮刷抖音的是不是底层？你以为农民和工人都不上网，都不看视频的？

你去看看这些人的感想吧，虽说有些幼稚、偏激，但是今年一年，被铁拳砸的，已经快怀疑人生了

**Reply by oldCassadin** [reply]
> > 他的意思是，颠覆性的力量只能来自于那些吃不饱穿不暖，更没有功夫上网的人。 有功夫在这叭叭的，通常被引导，分化，最终都会轻易的妥协，没有几个人敢真的豁出去的。
> 
> 你去看看这两月，抖音、头条、西瓜视频上，各种评论是怎么翻车的。
> 
> 这帮刷抖音的是不是底层？你以为农民和工人都不上网，都不看视频的？
> 
> 你去看看这些人的感想吧，虽说有些幼稚、偏激，但是今年一年，被铁拳砸的，已经快怀疑人生了

让我看没有用，因为我不是切蛋糕的人。
上述核心是吃不饱穿不暖，不要转移到农民工人不上网。  
农民/工人≠过的很差，更不等于底层。
买到烂尾楼是很可怜，但有更更更更多，这辈子攒到死连首付门槛都摸不到从来不敢奢望自己能买房的人。
btw，我只是揣测并尝试解释一下楼主的想法，可能对可能不对，利益不相关，不站任何立场（狗头）.

**Comment by lisongrui** [comment]
武汉光谷绿地星河绘项目的处置中 监管部门从该项目所剩无几的监管账户中 支出了一千多万 支付了施工单位拖欠的农民工工资 就看得出政府明白要稳住哪些人

**Comment by King8611** [comment]
比较认可你的观点，但又觉得你说的太直接了
